linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_silent/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol8
+prodtype: ol8,rhel8,rhel9
```

Comment:
I think you will have to assign CCEs to this rule

Reply:
It is done. I updated the commit where the rule was initially changed.